index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>Personal Portfolio</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1" />
+  <link rel="stylesheet" type="text/css" href="assets/css/style.css" />
+</head>
+
+<body>
+
+  <header>
+    <div class="wrapper menu">
+      <span class="menu-logo">My Logo</span>
+      <span class="menu-button"></span>
+    </div>
+  </header>
+
+  <main>
+
+    <section class="wrapper hero">
+      <h1 class="hero-title">Hey There. I’m Alex.</h1>
+      <div class="hero-subtitle">I'm a Software Developer</div>
+      <div class="hero-text">
+        I can help you build a product, feature or website Look through some of my work and experience!
+        If you like what you see and have a project you need coded, don’t hesitate to contact me.
+      </div>
+      <ul class="hero-social">
+        <li><a class="hero-social-link"><img height="20px" src="assets/svg/github.svg" alt="github" /></a></li>
+        <li><a class="hero-social-link"><img height="20px" src="assets/svg/linkedin.svg" alt="linkedin" /></a></li>
+        <li><a class="hero-social-link"><img height="20px" src="assets/svg/wellfound.svg" alt="wellfound" /></a></li>
+        <li><a class="hero-social-link"><img height="20px" src="assets/svg/twitter.svg" alt="twitter" /></a></li>
+        <li><a class="hero-social-link"><img height="20px" src="assets/svg/medium.svg" alt="medium" /></a></li>
+      </ul>
```

Comment:
Wonderful job on the "hero-social" icons list! The use of the `<ul>` tag to create a list of icons, with each icon wrapped in an `<a>` tag inside an `<li>` tag, is a great approach. It creates a clean and organized structure. :grinning:

However, to make the icons functional and improve user experience, it's important to provide appropriate href attributes for each `<a>` tag. The href attribute should point to the corresponding social media links, allowing users to navigate to the desired destinations when clicking on the icons.:grinning:

- [ ] Please ensure that you update the href attribute for each icon with the appropriate social media links. This will ensure that the icons are fully functional and provide a seamless user experience.:pray: